backend/tests/expenses/test_expense_routes.py

```
@@ -1,6 +1,7 @@
 from unittest.mock import AsyncMock, patch
 
 import pytest
+from app.auth.dependencies import get_current_user
```

Ruff (0.12.2)

`app.auth.dependencies.get_current_user` imported but unused

Remove unused import: `app.auth.dependencies.get_current_user`

(F401)